crates/sui-protocol-config/src/lib.rs

```diff
@@ -530,6 +534,13 @@ pub struct ProtocolConfig {
     hmac_hmac_sha3_256_cost_base: Option<u64>,
     hmac_hmac_sha3_256_input_cost_per_byte: Option<u64>,
     hmac_hmac_sha3_256_input_cost_per_block: Option<u64>,
+
+    // ==== Configs introduced in V2 ====
```

Comment:
V3? But I continue to think that we should not have sections for these by-version-number and instead you should just put next to the other related limits in the struct.